iocore/net/SSLUtils.cc

```diff
@@ -1629,7 +1629,8 @@ SSLInitServerContext(const SSLConfigParams *params, const ssl_user_config *sslMu
     }
 
     if (!sslMultCertSettings->cert) {
-      if (sslMultCertSettings->opt != SSLCertContext::OPT_TUNNEL) {
+      // There must be a certificate specified unless the tunnel action is set, or this is the default case
+      if (sslMultCertSettings->opt != SSLCertContext::OPT_TUNNEL && strcmp(sslMultCertSettings->addr, "*") != 0) {
```

Comment:
In this case could "*" be the default destination configured by the operator? Is that required to have a certificate?

Maybe this validation should be done after parsing a config file entry?